src/tools/perf/perftest.c

```diff
@@ -1357,13 +1357,27 @@ static ucs_status_t check_system(struct perftest_context *ctx)
         }
         CPU_SET(ctx->cpu, &cpuset);
 
+#if defined(HAVE_SCHED_SETAFFINITY)
         ret = sched_setaffinity(0, sizeof(cpuset), &cpuset);
+#elif defined(HAVE_CPUSET_SETAFFINITY)
+	ret = cpuset_setaffinity(CPU_LEVEL_WHICH, CPU_WHICH_PID, getpid(),
```

Comment:
maybe add ucs wrappers from set/get affinity in ucs/sys/sys.c
it's used in multiple places
(+ fix indentation)